### PR TITLE
Guide to disussions and issues in README.md

### PR DESCRIPTION
**Pull request by sirhcel**
Several questions have ended up in the issues recently. So what about pointing the discussions section out more prominently?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
-> Discover the world of microcontrollers through Rust
+Discover the world of microcontrollers through
+[Rust](https://www.rust-lang.org/):
+
+- Read this book on-line at https://docs.rust-embedded.org/discovery/
+- Start working on the examples from this repository
+- You've got questions?
+    - Have a look at our [discussions section on
+      GitHub](https://github.com/rust-embedded/discovery/discussions)
+    - May be it got already answered there
```
Comment (adamgreig) [inline README.md:11]:
```suggestion
    - Maybe it has already been answered
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+    - If not, start a new discussion
+- You've found an issue?
+    - Have a look at our [issues on
+      GitHub](https://github.com/rust-embedded/discovery/issues)
+    - May be there is already a workaround
```
Comment (adamgreig) [inline README.md:16]:
```suggestion
    - Maybe there is already a workaround
```

**Review by adamgreig — COMMENTED** [review]
Thanks, this is a great idea! I've made a couple of small wording changes which I'll commit and then get this merged.

**Review by adamgreig — APPROVED** [review]
bors merge

**Comment by sirhcel** [comment]
Thank you for cleaning up and fixing my spelling! :)

**Pull request merged**